### DISCUSSION TITLE
How to store a single "\"

### DISCUSSION THREAD

**Discussion by geenweb** [nlohmann/json#4030] [Q&A] [answered]
**Close this question.**
 I see that it saves as "\\\\", but when it reads in the value is "\\". I assume this must be JSON standard.
-Gene

---------------------------------------------------------------------------------------------------------------------------------------------------
I need to have a single backslash as a value in a JSON file. When the following line of code is executed:
`data["key_fields"][0]["1"]["char"] = "\";`

It creates:
```
"key_fields": [
  {
    "1": {
      "char": "\\"
```
Is there any workaround for this?

Thanks

Gene

**Comment by nlohmann** [comment, marked as answer]
This is correct since backslashes need to be escaped. Try:

```cpp
std::string slash = data["key_fields"][0]["1"]["char"];
std::cout << slash << std::endl;
```
